tests/checks/.IGNORE-PODMAN

```diff
@@ -0,0 +1,8 @@
+# TESTS TO BE IGNORED WHEN PODMAN
+# IS NOT AVAILABLE
+ampa.nf
+blast-parallel.nf
+blast.nf
+workdir-with-blank.nf
+stress.nf
+env-container.nf
```

Comment:
This can be removed.

Reply:
removed file via commit fdae5e1fcc243962a79c284ff29d282c793349cd